src/appleseed/foundation/meta/benchmarks/benchmark_basis.cpp

```diff
@@ -0,0 +1,133 @@
+
+//
+// This source file is part of appleseed.
+// Visit http://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2010-2013 Francois Beaune, Jupiter Jazz Limited
```

Comment:
If this is a new file (it appears so), you should replace these two lines by a single one with your name:

    // Copyright (c) 2017 Aytek Aman, The appleseedhq Organization

The rules are explained here: https://github.com/appleseedhq/appleseed/wiki/Copyright-Notices